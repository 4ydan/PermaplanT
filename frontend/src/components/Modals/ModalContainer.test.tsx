import renderer from 'react-test-renderer';
import ModalContainer from './ModalContainer';
import {MemoryRouter} from 'react-router-dom';

it('renders correctly', () => {
    const tree = renderer
        .create(
            <MemoryRouter>
                    <ModalContainer children={[]} show={true} />
            </MemoryRouter>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});