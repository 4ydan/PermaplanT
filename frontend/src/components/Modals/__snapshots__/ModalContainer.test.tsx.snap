// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="fixed top-[15%] left-[50%] z-[1020]"
  style={
    {
      "opacity": 0,
      "transform": "translateX(-50%) translateZ(0)",
    }
  }
/>
`;
